buildSrc/src/main/rocker/com/uber/okbuck/template/android/ResourceRule.rocker.raw

```diff
@@ -24,6 +25,13 @@ boolean resourceUnion)
     }
     ]),
 }
+@if (valid(exportedDeps)) {
```

Comment:
does this require a buck change?

Reply:
yes, requires a buck change (have disabled the functionality in the okbuck extension). will enable once OkBuck is updated. Linked the buck PR in the description.